notes/views/test_rca.py

```
@@ -0,0 +1,29 @@
+    
+def test_keyerror(user_data):
+    rewards = user_data.get("days", None)
+    if rewards > 30:
+        print("user is eligible")
+        return True
+    return False
+
+def main_func(data):
+    if test_rewards():
+        return True
+    return False
```

Ruff

Return the condition `bool(test_rewards())` directly

Replace with `return bool(test_rewards())`

(SIM103)